core/block/editor/clipboard/paste.go

```diff
@@ -290,9 +291,18 @@ func (p *pasteCtrl) insertUnderSelection() (err error) {
 	})
 }
 
+func isObligatoryRelation(targetID string) bool {
```

Comment:
Imho, `Required` is more common word than `Obligatory`

Reply:
so be it